packages/giselle-engine/src/core/generations/telemetry.ts

```diff
@@ -120,7 +120,12 @@ export function createLangfuseTracer({
 	const trace = langfuse.trace({
 		userId: String(settings?.metadata?.userId),
 		name: spanName,
-		metadata: settings?.metadata,
+		metadata: {
+			...settings?.metadata,
+			...(process.env.VERCEL_DEPLOYMENT_ID && {
+				deploymentId: process.env.VERCEL_DEPLOYMENT_ID,
```

Comment:
we use raw Vercel deployment ID here.
Thank you for your comment on #890 @shige 

related comment: https://github.com/giselles-ai/giselle/pull/890#discussion_r2101561464